config/routes.rb

```diff
@@ -1,17 +1,18 @@
 Rails.application.routes.draw do
 
   root to: 'home#index'
-  resources :cards do
-    post :review, on: :member
-  end
+  resources :cards
   resources :reviews, only: [:new, :create]
+  resources :decks do
+    get :set_current, on: :member
+  end
   resources :registrations, only: [:new, :create]
   resources :user_sessions, only: [:new, :create, :destroy]
   resource  :profile, controller: :profile, only: [:edit,:update]
   
   get  'sign_up' => 'registrations#new',     :as => :signup
   get  'login'   => 'user_sessions#new',     :as => :login
-  post 'logout'  => 'user_sessions#destroy', :as => :logout
+  delete 'logout'  => 'user_sessions#destroy', :as => :logout
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.